TrafficCapture/dockerSolution/src/main/docker/migrationConsole/kafkaExport.sh

```diff
@@ -60,6 +74,20 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
+if [ -n "$partition_offsets" ]; then
+    # Prepend the topic name to each partition offset
+    partition_offsets_with_topic=$(echo "$partition_offsets" | awk -v topic="$topic" 'BEGIN{RS=",";ORS=","}{print topic ":" $0}' | sed 's/,$//')
+else
+    partition_offsets_with_topic=""
+fi
+
+if [ -n "$partition_limits" ]; then
+    # Prepend the topic name to each partition limit
+    partition_limits_with_topic=$(echo "$partition_limits" | awk -v topic="$topic" 'BEGIN{RS=",";ORS=","}{print topic ":" $0}' | sed 's/,$//')
+else
+    partition_limits_with_topic=""
+fi
+
 partition_offsets=$(./kafka/bin/kafka-run-class.sh kafka.tools.GetOffsetShell --broker-list "$broker_endpoints" --topic "$topic" --time -1 $(echo "$kafka_command_settings"))
 comma_sep_partition_offsets=$(echo $partition_offsets | sed 's/ /,/g')
```

Comment:
As far as I can tell, we're now ignoring these partition offsets in favor of `partition_offsets_with_topic`.
First of all, is it correct that we're now ignoring them? If so, let's remove this code.

Reply:
These have always just been a print statement which allows a user to understand what the offset on the replayer may be, i've updated the variable names to make that more clear